tools/depends/configure.ac

```diff
@@ -266,6 +270,13 @@ case $host in
         if test "x$use_cpu" = "xauto"; then
           use_cpu=$host_cpu
         fi
+        use_cmake_cpu=i686
+      ;;
+      x86_64*-linux-android*)
+        if test "x$use_cpu" = "xauto"; then
+          use_cpu=x86_64
+        fi
+        use_cmake_cpu=x86_64
```

Comment:
This `x86_64*-linux-android*)` was missing before

Reply:
How is adding a new platform which currently isn't support, related to this PR?

Reply:
Becomes separated, but makes no sense to define a i686 for Android only and no x86_64 where is mostly the standard of them (by x86 CPU way)

And https://github.com/xbmc/xbmc/blob/master/docs/README.Android.md not correct, this makes on my system a 64bit and if I take a very old PC a 32bit

Reply:
Where does the the android documentation mention a x86_64 host?
How is your x86_64 build platform related to the different build platforms?

Reply:
One is here https://gitlab.kitware.com/cmake/cmake/blob/v3.13.0/Modules/Platform/Android-Determine.cmake#L203, the `_ANDROID_ABI_x86_64_PROC` and `_ANDROID_ABI_x86_PROC`
And the https://developer.android.com/ndk/guides/standalone_toolchain 

Also how we use `host_cpu` and `./configure --host=...` seems not really correct. 
Host is for me my base where `x86_64` and normally by `configure` it should use `--target=...` (or I think bad :thinking:)